src/main/main.cpp

```diff
@@ -31,9 +33,65 @@
 #include <vector>
 
 using namespace clio;
+namespace po = boost::program_options;
+
+/**
+ * @brief Parse command line and return path to configuration file
+ *
+ * @param argc
+ * @param argv
+ * @return std::string Path to configuration file
+ */
+std::string
+parseCli(int argc, char* argv[])
+{
+    static constexpr char defaultConfigPath[] = "/etc/clio.json";
```

Comment:
Is this reasonable @legleux ?

Reply:
That's funny, I did mention that to him. :eyes:

Reply:
@godexsoft 
If this is the default place clio is going to look for a config, it should be `/etc/opt/clio/config.json` which is a link to `/opt/clio/etc/config.json`.
I can change the filename to whatever we want.

Reply:
Ok, changing it.